src/main/java/jaryl/duke/Output.java

```diff
@@ -0,0 +1,266 @@
+package jaryl.duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Output handles I/O, interactions with user
+ */
+public class Output {
+    /**
+     * Prints a line as a separator
+     * @return returns the line as a String
+     */
+    public String addLine() {
+        return "    ----------------------------------------------------------------------------------------------------------------------------------------------------------";
+    }
```

Comment:
Maybe you can consider making all the messages / strings named constants at the beginning of your class, so that the meaning of each string is well-explained, and it makes your code more succinct and readable as well.